### DISCUSSION TITLE
Local nominatim server is unable to find an address that nominatim.openstreetmap.org can find

### DISCUSSION THREAD

**Discussion by deKeijzer** [osm-search/Nominatim#3047] [Q&A]
Hi there, 

Hopefully, this is the right place for questions. 

I'm having some trouble with my local Nominatim server that I am running through 
```
docker run -it -e PBF_URL=https://download.geofabrik.de/europe/netherlands-latest.osm.pbf -e THREADS=32 -p 8080:8080 -v ~/nominatim-data:/data --name nominatim mediagis/nominatim:4.2
```
For some reason, it is unable to find an address which `nominatim.openstreetmap.org` is able to find.

The Docker should be running Nominatim `4.0.2.`. For some reason, I'm getting no results for

`http://IP:8080/search?q=Hilverbeekstraat 36, 1024, Amsterdam, Netherlands`

whereas I can find this address on nominatim.openstreetmap.org. 

Strangely some new addresses I can find perfectly fine using the local Nominatim server. Note that `Hilverbeekstraat 36, 1024, Amsterdam, Netherlands` is not necessarily a new address. I'm just amazed that my local server can find new addresses in the same city.

Did I miss something in the setup?

**Comment by lonvia** [comment]
If you are running on Nominatim 4.0, then you are likely still using the legacy tokenizer. This one cannot determine that `1024` is a superset of `1024KL`. Might be the problem here. But then again, the docker line claims to be running Nominatim 4.2.
